Benchmark with Apple Macbook Air M1

djawnstj opened the pull request:
Detail
======
 - #298
 - CPU: Apple Silicon M1
 - Memory: 8GB
 - OS: macOS Monterey v12.4
 - TypeScript-JSON version: 3.3.20

djawnstj committed:
Benchmark with Apple Macbook Air M1
Detail
======
 - #298
 - CPU: Apple Silicon M1
 - Memory: 8GB
 - OS: macOS Monterey v12.4
 - TypeScript-JSON version: 3.3.20

samchon reviewed (changes requested):
README is empty. Check it agin please.

djawnstj committed:
Benchmark with Apple Macbook Air M1
Detail
======
 - #298
 - CPU: Apple Silicon M1
 - Memory: 8GB
 - OS: macOS Monterey v12.4
 - TypeScript-JSON version: 3.3.20

djawnstj committed:
Benchmark with Apple Macbook Air M1
Detail
======
 - #298
 - CPU: Apple Silicon M1
 - Memory: 8GB
 - OS: macOS Monterey v12.4
 - TypeScript-JSON version: 3.3.20

samchon reviewed (approved):
Thanks for contribution ^^b
